src/pages/events/configure-commerce.md

```diff
@@ -30,20 +30,30 @@ You must configure Commerce to communicate with your project. Configuration incl
 
 1. Copy the contents of the `<workspace-name>.json` file into the **Adobe I/O Workspace Configuration** field.
 
-1. Enter a unique identifier in the **Adobe Commerce Instance ID** field. This value identifies your project and routes events from Adobe Commerce to the correct event provider in Adobe I/O. The value of this field is visible in App Builder during the creation of Event Registration.
+1. Enter a unique identifier in the **Adobe Commerce Instance ID** field. This value identifies your project and routes events from Adobe Commerce to the correct `Event Provider` in Adobe I/O. The value of this field is visible in App Builder during the [creation of Event Registration](#subscribe-and-register-events).
 
-   **NOTE**: The **Adobe Commerce Instance ID** field only support alphanumeric characters, hyphens and underscores.
+   **NOTE**: The **Adobe Commerce Instance ID** field only supports alphanumeric characters, hyphens and underscores.
 
 1. Click **Save Config**, but do not leave the page. The next section creates an event provider, which is necessary to complete the configuration.
 
```

Comment:
```suggestion
1. Click **Save Config**, but do not leave the page. The next section creates an event provider, which is necessary to complete the configuration.

 The event provider will not appear in the Developer Console until after you subscribe to a specific event.

```

Reply:
Does a "specific event" mean "any event" in this context? As it requires subscribing to any at least one event to make the Event Provider shown in UI

Reply:
i think so, @naomirobichaud is it any event?

Reply:
At least 1 event - we should specify that subscribe in this case means configuring an event to be emitted on the Commerce side so via io_events.xml, config.php, CLI